tutorial_5-BonusNotebook.ipynb

```diff
@@ -73,16 +73,9 @@
     "<details>\n",
     "  <summary>Click for help</summary>\n",
     "\n",
-    "```\n",
-    "mamba install -y -c conda-forge -c bioconda \\\n",
-    "    fastqc=0.11.9 \\\n",
-    "    multiqc=1.13 \\\n",
-    "    bismark=0.23.1 \\\n",
-    "    trim-galore=0.6.7 \\\n",
-    "    bedtools=2.30.0 \\\n",
-    "    samtools=1.15.1 \\\n",
-    "    metilene=0.2.8\n",
-    "```\n",
+    "\n",
+    "`mamba install -y -c conda-forge -c bioconda fastqc=0.11.9 multiqc=1.13 bismark=0.23.1 trim-galore=0.6.7 bedtools=2.30.0 samtools=1.15.1 metilene=0.2.8` \n",
```

Comment:
make sure to add how to add mamba to path before this: 
import os
os.environ["PATH"] += os.pathsep + os.environ["HOME"]+"/mambaforge/bin"

Reply:
That's been added.